x-pack/plugins/observability_solution/synthetics/common/constants/monitor_defaults.ts

```diff
@@ -115,8 +115,11 @@ export const PROFILES_MAP = PROFILE_VALUES.reduce((acc, profile) => {
   return acc;
 }, {} as { [key: string]: ThrottlingConfig });
 
+export const ALLOWED_SCHEDULES_IN_SECONDS = ['10s', '30s'];
+
 export const ALLOWED_SCHEDULES_IN_MINUTES = [
   '1',
+  '2',
```

Comment:
We need to make this change, both the allowed schedule in minutes and the allowed schedule in seconds, to the synthetics agent as well.

Reply:
that will be done in a follow up